### PR TITLE
Ci automation test

### PR DESCRIPTION
**Pull request by killmeplz**
Automated tag creation on new version of actions/runner

### CODE REVIEW

File: .github/workflows/updater.yml [hunk 1]
```diff
@@ -1,35 +1,64 @@
...
 jobs:
-  check-for-updates:
+  publish:
+    name: Publish new tag
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        versionRegex:
+         - v2\.[0-9]+\.[0-9]+$
+
```
Comment (dennybaa) [inline .github/workflows/updater.yml]: Great! Thank you @killmeplz , plz remove matrix, we don't plan support for multiple runner versions, the latest runner version is only relevant...

File: .github/workflows/updater.yml [hunk 2]
```diff
@@ -1,35 +1,64 @@
...
+        compared-to-tag: ${{ steps.upstream.outputs.tag }}
+        regex: ${{ matrix.versionRegex }}
+        repository: ${{ github.repository }}
 
-    - name: Create PR
-      uses: peter-evans/create-pull-request@v3
+    - name: Publish new release
+      uses: ncipollo/release-action@v1
+      if: steps.latest.outputs.newer == 'true' || steps.latest.outputs.tag == ''
       with:
```
Comment (dennybaa) [inline .github/workflows/updater.yml]: @killmeplz So could you please clarify... We create a github release, i.e. release and tag and what happens after this?

Comment (dennybaa) [reply .github/workflows/updater.yml]: @killmeplz I mean how do we publish the docker image and run that corresponding workflow?

**Review by dennybaa — CHANGES_REQUESTED** [review]

File: .github/workflows/updater.yml (hunk 2 above)
Comment (killmeplz) [reply .github/workflows/updater.yml]: 'Build and Push' workflow is gonna run on tag creation

Comment (dennybaa) [reply .github/workflows/updater.yml]: Yes, it's what I would think too. However 'Build and Push' won't run, we can't use recursive invocation of workflows in fact. So in fact only this workflow will run and others which might possibly triggered on release or tag will be pushed(

Comment (dennybaa) [reply .github/workflows/updater.yml]: If you'd like to check this @killmeplz, plz let me know and I'll approve and merge this, but please move the workflow into a separate file first, so we do not override the original update from the upstream. We are not interested in solving merge conflicts in the future.

**Comment by dennybaa** [comment]
@killmeplz Thank you, let's try like this

**Pull request merged**